modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.cc

```diff
@@ -742,6 +702,18 @@ LBSSolver::SetOptions(const InputParameters& params)
       }
     }
   } // for p
+
+  if (options_.write_restart_time_interval > 0)
```

Comment:
If `write_restart_path` is not specified, where would restarts be written to?

Reply:
Nice catch. I forgot to go back and set the default value for `write_restart_path`. I'll fix that.

Reply:
Fixed!